__tests__/integration.test.js

```diff
@@ -162,3 +162,79 @@ describe('/api/articles/:article_id/comments', () => {
   });
 
 });
+
+describe('POST /api/articles/:article_id/comments', () => {
+  test('POST 201: successfully inserts a new comment to the db, and returns article to client', () => {
+    const newComment = {
+      "username": "butter_bridge",
+      "body": "wow look, a new comment!"
+    }
+    return request(app)
+    .post('/api/articles/1/comments')
+    .send(newComment)
+    .expect(201)
+    .then((response) => {
+
+    expect(response.body.comment.author).toBe("butter_bridge"),
+    expect(response.body.comment.body).toBe('wow look, a new comment!')
+    })
+  });
```

Comment:
An extra test could be that it ignores unnecessary properties but still posts the comment if the correct ones are included